website/docs/connect-data/concepts/README.md

```diff
@@ -1,37 +1,51 @@
 # Concepts
 
 <div class="containerGridSampleApp">
-  <div class="containerColumnSampleApp columnGrid column-one">
+<div class="containerColumnSampleApp columnGrid column-one">
+        <div class="containerCol">
+            <a href="/connect-data/concepts/dynamic-binding-in-queries"><strong>Dynamic Binding in Queries</strong></a>
+        </div> <hr/>
+        <div class="containerDescription">Bind data dynamically to your queries in Appsmith.</div>
```

Comment:
Can you elaborate and come up with a better description?

Reply:
Done.